bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -44,4 +44,20 @@ components:
       test-configs:
         - with-security
         - without-security
+  - name: dashboards-reports
+    working-directory: reports-scheduler
```

Comment:
Correct me here for the approach. Do we need to specify the working directory? Can't we check for `integtest.sh` file in the plugin repo if not present then change directory to the plugin directory where `integtest.sh` is present?

Reply:
The `integtest.sh` are now removed from each plugin, we call the default `integtest.sh` which is present in the build repo. The working-directory is for it to understand where to run the `./gradlew integtest` command.

Reply:
Makes sense. Thanks for the clearing it out.